components/invitation_modal/invitation_modal_guests_step.jsx

```diff
@@ -58,18 +58,32 @@ export default class InvitationModalGuestsStep extends React.Component {
         this.props.onEdit(this.state.usersAndEmails.length > 0 || this.state.channels.length > 0 || e.target.value !== '');
     }
 
-    usersLoader = async (term) => {
+    debouncedSearchProfiles = debounce((term, callback) => {
```

Comment:
I'm not seeing callback being handled on the original code, wouldn't that mean that we might receive an undefined? if so, we should probably handle that, as we are calling it several times on the new code.

I might have missed it being handled, if so disregard the comment!

Reply:
Actually not, it is using react-select under the hood, and react-select expect a function that returns a promise, or handle it with the callback, each option is valid for react-select, so the change works.